docs/plugins.rst

```diff
@@ -166,6 +166,9 @@ plugin under consideration.  The objects themselves are relatively simple:
       enabled: false
       path: path/to/quux
 
+    - name: docs
+      enabled: true
+
```

Comment:
The plugin config file should still be a list of objects.  The only difference is that the ``path`` property is now optional (with different semantics when it is omitted).  Maybe I misunderstood your comment?